### ISSUE TITLE
Issue when trying to bundle

### ISSUE THREAD

**Issue by CrashyBang** [screendriver/convert-vapid-public-key#44]
When trying to bundle I receive the following error:
```
./toUint8Array in ./node_modules/convert-vapid-public-key/dist/src/index.js
```

Cheers.

**Comment by screendriver** [comment]
Could you please provide more context? 

- which bundler? 
- which version? 
- which error message? (you didn't provide the error message unfortunately) 
- general more data about your environment 

Thanks :+1:

**Comment by CrashyBang** [comment]
Hey @screendriver,

Sorry some of the error got cut off :L I will attach it tomorrow at work.

Using webpack (with `vue-cli`).
Ubuntu 18.10.
Node 10.x

Let me know if you need anything else, cheers.

**Comment by screendriver** [comment]
Ah I see. I forgot to deploy that file 🤦‍♂ 

I quickly released `v1.0.1` that should include everything. Thank you for your report.
